src/dom/dom.js

```diff
@@ -1704,8 +1704,13 @@ p5.Element.prototype.html = function() {
  * href="https://developer.mozilla.org/en-US/docs/Web/CSS/position">positioning scheme</a>.
  * If no arguments given, the function returns the x and y position of the element.
  *
+ * found documentation on how to be more specific with object type
+ * https://stackoverflow.com/questions/14714314/how-do-i-comment-object-literals-in-yuidoc
```

Comment:
I would prefer if this was left out from the codebase but mentioned in the PR text or the commit message.